src/utilities/buildASTSchema.js

```diff
@@ -466,6 +447,42 @@ export function buildASTSchema(ast: DocumentNode): GraphQLSchema {
 }
 
 /**
+ * This takes the ast of a schema document produced by the parse function in
+ * src/language/parser.js.
+ *
+ * If no schema definition is provided, then it will look for types named Query
+ * and Mutation.
+ *
+ * Given that AST it constructs a GraphQLSchema. The resulting schema
+ * has no resolve methods, so execution will use default resolvers.
+ */
+export function buildASTSchema(ast: DocumentNode): GraphQLSchema {
+  const config:any = buildASTSchemaConfig(ast);
```

Comment:
@wincent I tried to make `config: GraphQLSchemaComponents` but it sims that Flow don't recognize runtime checks like this one: https://github.com/graphql/graphql-js/pull/817/files#diff-2691b670eb7e51187edc1f918c624a70R468
So when I call `new GraphQLSchema(config)` flow produce an error saying `query` is missing. Can you please advice on how to make this possible in Flow?